app.js

```diff
@@ -139,7 +139,10 @@ define(function(require) {
 			var self = this,
 				container = parent.find('.right-content');
 
-			parent.find('.left-menu').on('click', '.category:not(.loading)', function() {
+			monster.util.isFeatureAvailable('smartpbx.other.eventRouting') ? window.addEventListener('message', function(event) {
```

Comment:
Looks good! I would only suggest to add a line break, to try to keep the line length within 80 characters as much as possible:
```suggestion
			monster.util.isFeatureAvailable('smartpbx.other.eventRouting') 
				? window.addEventListener('message', function(event) {
```